docker-cloud/cloud-swarm/create-cloud-swarm-aws.md

```diff
@@ -0,0 +1,113 @@
+
+    Alternatively, you can select **+ -> Swarm** from the top navigation to get to the same page.
+
+3.  Enter a name for the new swarm.
+
+    Your Docker ID is pre-populated. In the example, our swarm name
+    is "vote-swarm".
+
+    ![](images/aws-create-swarm-1-name.png)
+
+    >**Tip:** For Docker Cloud, use all lower case letters for swarm names. No spaces, capitalized letters, or special characters other than `.`, `_`, or `-` are allowed. AWS does not accept underscores in the name `_`.
+
+4.  Select Amazon Web Services as the service provider, select a channel (`Stable` or `Edge`) from the drop-down menu, and provide an App name.
+
+    In this example, we'll use the `Stable` channel, and use "voting_app" as our application name.
+
+    ![](images/aws-create-swarm-0.png)
+
+5.  Select a **Region** from the drop-down menu.
+
+    > **Tip:** The SSH keys available to you in the next steps are
+    filtered by the region you select here. Make sure that you have
+    appropriate SSH keys available on the region you select.
+
+    Optionally, click **Region Advanced Settings** to configure a
+    [Virtual Private Cloud(VPC)](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Introduction.html) on which to run this swarm.
+
+    ![](images/aws-create-swarm-3-region.png)
+
+    For guidance on setting up a VPC, see [Recommended VPC and subnet setup](https://docs.docker.com/docker-for-aws/faqs/#can-i-use-my-existing-vpc) in the Docker for AWS topics.
+
+6.  Choose how many swarm managers and swarm worker nodes to deploy.
+
+    Here, we create one manager and two worker nodes. (This maps nicely to the [Swarm tutorial setup](/engine/swarm/swarm-tutorial/index.md) and the [voting app sample in Docker Labs](https://github.com/docker/labs/blob/master/beginner/chapters/votingapp.md).)
+
+    ![](images/cloud-create-swarm-4-size.png)
+
+7.  Configure swarm properties.
+
+    ![](images/aws-create-swarm-5-properties.png)
+
+    * Select a public SSH key for Docker Cloud to use to connect to the
+    nodes on AWS. Public keys from the [key pairs you configured on AWS](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) are provided in the drop-down menu. Only keys associated with the
+    Region you selected (in step 5) are shown.
+
+    * Choose whether to provide daily resource cleanup.
+
+    * Enable or disable Cloudwatch for container logging. (It is enabled by default.)
+
+    <font style="color:red;">
+    RYAN, ALBERTO:<br>
+    Can you provide more detail here on what daily resource cleanup does and pros and cons of having this on/off?<br><br>
+    Same for Cloudwatch for container logging, can you provide more detail?
+    </font>
+
+7. Select the instance sizes for the managers, and for the workers.
+
+    ![](images/aws-create-swarm-6-manager-worker.png)
+
+    <font style="color:red;">RYAN, ALBERTO: Should we add more detail on machine sizes?</font>
```

Comment:
Here is a link to the instance options and pricing on Azure: https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/

Perhaps we could include a link.